frontend/app/component/form/candidates_votes_form/CandidatesVotesForm.tsx

```diff
@@ -166,11 +166,14 @@ export function CandidatesVotesForm({ group }: CandidatesVotesFormProps) {
           />
         </InputGrid.Body>
       </InputGrid>
-      <BottomBar type="inputgrid">
+      <BottomBar type="input-grid">
         <Button type="submit" size="lg" disabled={loading}>
           Volgende
         </Button>
-        <span className="button_hint">SHIFT + Enter</span>
+        <span className="button_hint">
+          <Shift />
+          <Enter />
+        </span>
```

Comment:
What do you think of combining all of this into a single 'keyboard keys' component?

Reply:
Might be a good idea, I expect there will be more types of keyboard in the future